<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosGaugedHydrographWidget</class>
 <widget class="QWidget" name="ReosGaugedHydrographWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current gauged hydrograph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComboBoxHydrographName"/>
     </item>
     <item>
      <widget class="QWidget" name="mWidgetToolBar" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ReosPlotWidget" name="plotWidget" native="true"/>
     <widget class="QWidget" name="mEditingWidget" native="true">
      <layout class="QVBoxLayout" name="mEditingWidgetLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReosPlotWidget</class>
   <extends>QWidget</extends>
   <header>reosplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
